express.js

```diff
@@ -17,9 +17,10 @@ app.locals.clanInvitations = {};
 require('dotenv').config();
 //Define environment variables with default values
 process.env.NODE_ENV = process.env.NODE_ENV || 'production';
-process.env.PORT = process.env.PORT || '4000';
+process.env.PORT = process.env.PORT || '3000';
```

Comment:
in every .env file i could found we setting the `PORT ` env, just because we have this fallback here.